cmd/cmd.go

```diff
@@ -0,0 +1,176 @@
+// Package cmd - package
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github/Asac2142/go-cli-app/file"
+	"github/Asac2142/go-cli-app/task"
+)
+
+const (
+	add            string = "add"
+	update         string = "update"
+	deleteTask     string = "delete"
+	markInProgress string = "mark-in-progress"
+	markDone       string = "mark-done"
+	list           string = "list"
+)
+
+// HandleTrackerCLI - Handles user's input
+func HandleTrackerCLI(f *file.File[task.TContent]) {
+	task := task.NewTask(f)
+	args := os.Args[1:]
+
+	if len(args) == 0 {
+		err := errors.New("Invalid input")
+		printError(err)
+	}
+
+	switch args[0] {
+	case add:
+		err := addTask(&args, task)
+		printError(err)
+	case update:
+		err := updateTask(&args, task)
+		printError(err)
+	case deleteTask:
+		err := removeTask(&args, task)
+		printError(err)
+	case markInProgress:
+		err := updateStatus(&args, markInProgress, task)
+		printError(err)
+	case markDone:
+		err := updateStatus(&args, markDone, task)
+		printError(err)
+	case list:
+		err := listTasks(&args, task)
+		printError(err)
+	default:
+		err := errors.New("Invalid task command")
+		printError(err)
+	}
+}
+
+func addTask(args *[]string, t *task.Task) error {
+	if len(*args) == 1 {
+		return errors.New("add task requires a description")
+	}
+
+	desc := (*args)[1]
+	added, err := t.Add(desc)
+
```

Comment:
It's idiomatic to not add empty line between the command returning error and if checking the error.